.travis.yml

```diff
@@ -1,30 +1,17 @@
 language: ruby
-os: osx
-env: OSX=10.11
-osx_image: xcode7.3
-rvm: system
+rvm: 2.0.0
+os: linux
```

Comment:
@UniqMartin In the past I've seen sporadic duplication of jobs that went away when I moved to this format.